packages/react-window-provider/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fluentui/react-window-provider",
-  "version": "0.3.3",
+  "version": "2.0.0-beta",
```

Comment:
bumping this one by 2 because we should have this versioned to be `1.0.0` to begin with since it's used in production. I will update this to `1.0.0` in `7.0.0` branch separately